### PR TITLE
Adding new iOS Tutorial

### PR DESCRIPTION
**Pull request by BobaFetters**
Created new iOS tutorial using version 1.2.0 and SwiftUI

Closes https://github.com/apollographql/team-prometheus/issues/72

### CODE REVIEW

File: docs/source/tutorial/tutorial-define-additional-mutations.mdx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+Now, click the Submit Operation button to run your authorized query. You'll get back information regarding the trips (or in this case, trip) you've just booked.
+
+> Note: If you receive an error that says "Cannot read property 'id' of null", that means your user was not found based on the token you passed through. Make sure your authorization header is properly formatted and that you're actually logged in!
+
+<img alt="Explorer showing the result of booking a trip with an array of IDs" class="screenshot" src="images/explorer_book_with_trip_ids.png"/>
+
+With a mutation written like this, you can book any number of trips you want at the same time. However, the booking mechanism in our application will only let you book one trip at a time.
+
+Luckily, there's an easy way to update the mutation so it's required to only take a single object. First, update the name of your operation in Explorer to the singular `BookTrip` (and remove `Mutation` since that will be added for us by code generation). Next, update the mutation to take a single `$id`, then pass an array containing that `$id` to the `bookTrips` mutation:
```
Comment (calvincestari) [inline docs/source/tutorial/tutorial-define-additional-mutations.mdx:43]: ~(and remove `Mutation` since that will be added for us by code generation)~ - this is implementation detail they really don't need to be concerned about. I believe we check for the word "Mutation" and append it if it's not there but either way if we have a bug that results in "BookTripsMutationMutation" then we should fix it.

tl;dr - this isn't tutorial-level detail that helps them learn Apollo iOS.

File: docs/source/tutorial/tutorial-subscriptions.mdx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+<img alt="Subscription notification view" class="screenshot" src="images/simulator_subscription_notification_view.png" width="300"/>
+
+Congratulations, you've completed the tutorial!
+
+## More resources
+
+There are way more things you can do with the Apollo iOS SDK, and the rest of this documentation includes info on more advanced topics like:
+
+- Using [fragments](../fragments/)
+- Working with [custom scalars](../fetching-queries/#notes-on-working-with-custom-scalars)
```
Comment (calvincestari) [inline docs/source/tutorial/tutorial-subscriptions.mdx:172]: I was going to suggest adding in a step about working with custom scalars if there is a field in the schema that could support it. Mentioning it here though might be enough for now.

**Review by calvincestari — APPROVED** [review]
This is great, thank you @BobaFetters! It would be great to get @StephenBarlow's wordsmith-ing skills for the general language of the tutorial if he has the time.

File: docs/source/tutorial/tutorial-introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* Run code generation
+* Write queries and mutations
+* Handle errors
+* Handle Pagination
+* Authenticate with a backend server
+* Use subscriptions to receive real-time updates
+
+The tutorial uses an instance of [Apollo Server](https://www.apollographql.com/docs/apollo-server/) hosted on Heroku, and an [Apollo Studio Sandbox](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fapollo-fullstack-tutorial.herokuapp.com%2Fgraphql&_gl=1*d2b33m*_ga*MzQ1Mzc2MDg1LjE2ODE3NjA2ODc.*_ga_0BGG5V2W2K*MTY4Mjk2MjIwMi4yNS4xLjE2ODI5NjIyMzIuMC4wLjA.) which connects to that server. If you're curious about how to build your own server, see the [Apollo full-stack tutorial](https://www.apollographql.com/docs/tutorial/introduction/).
+
+> If you encounter any issues during the tutorial, feel free to ask questions by either [opening an issue on our GitHub repo](https://github.com/apollographql/apollo-ios/issues) or [stopping by our community forums for help](https://community.apollographql.com).
```
[the remark this answers is not in the document]
Comment (BobaFetters) [reply docs/source/tutorial/tutorial-introduction.mdx:17]: Hmmm that's a good question, originally the tutorial was pointing to the apollo-ios repo, but I think it may be better to direct issues to the iOSTutorial repo since any issue that comes up is likely directly related to the tutorial.

[the remark this answers is not in the document]
Comment (calvincestari) [reply docs/source/tutorial/tutorial-introduction.mdx]: I think so too.

File: docs/source/config.json [hunk 4]
```diff
@@ -10,7 +10,23 @@
     "Migrating to v1.0": "/migrations/1.0",
     "Project Configuration": "/project-configuration",
     "Tutorial": {
-      "Code Generation": "/tutorial/codegen-getting-started"
+      "Code Generation": "/tutorial/codegen-getting-started",
+      "Build a project with Apollo": {
```
[the remark this answers is not in the document]
Comment (BobaFetters) [reply docs/source/config.json:14]: Would be curious about this as well, from what I see it looks like this is how it is for others (Kotlin etc) as well, but if there is a way to do it that would be nice.

File: docs/source/config.json [hunk 5]
```diff
@@ -10,7 +10,23 @@
     "Migrating to v1.0": "/migrations/1.0",
     "Project Configuration": "/project-configuration",
     "Tutorial": {
-      "Code Generation": "/tutorial/codegen-getting-started"
+      "Code Generation": "/tutorial/codegen-getting-started",
```
Comment (AnthonyMDev) [inline docs/source/config.json:13]: This sidebar looks kind of awkward to me now in the deploy preview. We have a full "build a sample application tutorial as well as a step-by-step guide for setting up code generation in your own project. 

We also have an entire section on Code Generation itself. I liked having the code generation tutorial separated from the conceptual documentation on code gen, but I'm not sure how to best organize this now. @StephenBarlow any suggestions?

File: docs/source/config.json [hunk 6]
```diff
@@ -10,7 +10,23 @@
...
+      "Build a project with Apollo": {
+        "0. Introduction": "/tutorial/tutorial-introduction",
```
Comment (AnthonyMDev) [inline docs/source/config.json:15]: I would remove the "0." here. "Introduction" and then step 1 makes sense to me.

**Review by AnthonyMDev — APPROVED** [review]
This looks amazing @BobaFetters! Thanks so much for your hard work on this!
